Session/Closed Groups/NewClosedGroupVC.swift

```diff
@@ -155,6 +159,11 @@ final class NewClosedGroupVC: BaseVC, UITableViewDataSource, UITableViewDelegate
         // Set up content
         setUpViewHierarchy()
     }
+    
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
+        tableViewWidth?.constant = size.width
+    }
```

Comment:
We don't seem to actually set the `tableViewWidth` variable anywhere - can we just remove the changes to this file?

Reply:
Removed the code, we don't need it anymore.